packages/ag-charts-community/src/scene/shape/text.ts

```diff
@@ -176,7 +255,7 @@ export class Text<D = any> extends Shape<D> {
 
     setFont(props: TextSizeProperties) {
         this.fontFamily = props.fontFamily;
-        this.fontSize = props.fontSize;
+        this.fontSize = props.fontSize ?? 10;
```

Comment:
Nit: Is this fallback strictly necessary? If so, could we extract a constant so it always aligns with the default for `this.fontSize`?

Reply:
Updated